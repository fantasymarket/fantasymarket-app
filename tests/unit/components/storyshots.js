import ReactDOM from 'react';
import initStoryshots from '@storybook/addon-storyshots';
import { render } from '@testing-library/react';

beforeAll(() => {
	ReactDOM.createPortal = jest.fn(element => {
		return element;
	});
});

initStoryshots({
	suite: 'Component Snapshots',
	configPath: './.storybook',
	renderer: render,
});
