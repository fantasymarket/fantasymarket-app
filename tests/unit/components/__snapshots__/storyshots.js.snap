// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button With Emoji 1`] = `
<button
  className="sc-AxjAm gXGLhX"
>
  <span
    aria-label="so cool"
    role="img"
  >
    üòÄ üòé üëç üíØ
  </span>
</button>
`;

exports[`Storyshots Button With Text 1`] = `
<button
  className="sc-AxjAm gXGLhX"
>
  Hello Button
</button>
`;

exports[`Storyshots Icon With Color 1`] = `
<div
  className="sc-AxirZ koClct"
  color="white"
>
  <svg
    fill="none"
    height="1em"
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 24 24"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="12"
      r="3"
    />
    <path
      d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
    />
  </svg>
</div>
`;

exports[`Storyshots Icon With SVG 1`] = `
<div
  className="sc-AxirZ koClct"
>
  <svg
    fill="none"
    height="1em"
    stroke="currentColor"
    strokeLinecap="round"
    strokeLinejoin="round"
    strokeWidth="2"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 24 24"
    width="1em"
    xmlns="http://www.w3.org/2000/svg"
  >
    <circle
      cx="12"
      cy="12"
      r="3"
    />
    <path
      d="M19.4 15a1.65 1.65 0 0 0 .33 1.82l.06.06a2 2 0 0 1 0 2.83 2 2 0 0 1-2.83 0l-.06-.06a1.65 1.65 0 0 0-1.82-.33 1.65 1.65 0 0 0-1 1.51V21a2 2 0 0 1-2 2 2 2 0 0 1-2-2v-.09A1.65 1.65 0 0 0 9 19.4a1.65 1.65 0 0 0-1.82.33l-.06.06a2 2 0 0 1-2.83 0 2 2 0 0 1 0-2.83l.06-.06a1.65 1.65 0 0 0 .33-1.82 1.65 1.65 0 0 0-1.51-1H3a2 2 0 0 1-2-2 2 2 0 0 1 2-2h.09A1.65 1.65 0 0 0 4.6 9a1.65 1.65 0 0 0-.33-1.82l-.06-.06a2 2 0 0 1 0-2.83 2 2 0 0 1 2.83 0l.06.06a1.65 1.65 0 0 0 1.82.33H9a1.65 1.65 0 0 0 1-1.51V3a2 2 0 0 1 2-2 2 2 0 0 1 2 2v.09a1.65 1.65 0 0 0 1 1.51 1.65 1.65 0 0 0 1.82-.33l.06-.06a2 2 0 0 1 2.83 0 2 2 0 0 1 0 2.83l-.06.06a1.65 1.65 0 0 0-.33 1.82V9a1.65 1.65 0 0 0 1.51 1H21a2 2 0 0 1 2 2 2 2 0 0 1-2 2h-.09a1.65 1.65 0 0 0-1.51 1z"
    />
  </svg>
</div>
`;

exports[`Storyshots Input With Emoji 1`] = `
<div
  className="sc-AxiKw gXLmey"
>
  <input
    value="üòÄ üòé üëç üíØ"
  />
</div>
`;

exports[`Storyshots Input With Label 1`] = `
<div
  className="sc-AxiKw gXLmey"
>
  <input />
</div>
`;

exports[`Storyshots Input With Placeholder 1`] = `
<div
  className="sc-AxiKw gXLmey"
>
  <input />
</div>
`;

exports[`Storyshots Input With Text 1`] = `
<div
  className="sc-AxiKw gXLmey"
>
  <input />
</div>
`;

exports[`Storyshots Tab With Color 1`] = `
<div
  className="sc-AxhCb cbHbNo"
>
  <button
    className="sc-AxhUy iHGQqu"
    onClick={[Function]}
    type="button"
  >
    TAB 1
  </button>
  <button
    className="sc-AxhUy heaGUV"
    onClick={[Function]}
    type="button"
  >
    TAB 2
  </button>
</div>
`;
